docker-nginx/pages/403.html

```diff
@@ -4,12 +4,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="/pages/favicon.ico" type="image/x-icon">
```

Comment:
I think we should fix the `type` in some other places in the codebase, e.g. on app.qfield.cloud , feel free to take over the follow up after this PR :)